closure/goog/events/inputhandler.js

```diff
@@ -81,7 +81,7 @@ goog.events.InputHandler = function(element) {
   //   event.
   // IE9 supports input events when characters are inserted, but not deleted.
   // WebKit before version 531 did not support input events for textareas.
-  var emulateInputEvents = goog.userAgent.IE ||
+  var emulateInputEvents = goog.userAgent.IE || goog.userAgent.EDGE ||
```

Comment:
Can you change this so that this is on by default and off for old IE and old Safari.